docs/src/main/asciidoc/cdi-reference.adoc

```diff
@@ -251,6 +251,16 @@ class MyBeanStarter {
 
 NOTE: Quarkus users are encouraged to always prefer the `@Observes StartupEvent` to `@Initialized(ApplicationScoped.class)` as explained in the link:lifecycle[Application Initialization and Termination] guide. 
 
+=== Request Context Activation 
```

Comment:
`Activation` -> `Automatic Activation`? (under current title I imagine a paragraph on how I manually activate it on demand)
It might also be worth noting that this will trigger all respective req. context lifecycle event such as firing `@Initilized`.

Reply:
`Automatic Activation` sounds odd to me. Maybe we could just copy the wording from the spec: `Request Context Lifecycle`?

> It might also be worth noting that this will trigger...

Yes, we could add a note.